time stamp
int32 robot_index
string status
string STALLED="STALLED"